hermes-management/src/main/java/pl/allegro/tech/hermes/management/infrastructure/zookeeper/ZookeeperRepositoryManager.java

```diff
@@ -61,6 +63,7 @@
     private final Map<String, UndeliveredMessageLog> undeliveredMessageLogsByDc = new HashMap<>();
     private final Map<String, AdminTool> adminToolByDc = new HashMap<>();
     private final Map<String, ReadinessRepository> readinessRepositoriesByDc = new HashMap<>();
+    private final Map<String, ZookeeperOfflineRetransmissionRepository> offlineRetransmissionRepositoriesByDc = new HashMap<>();
```

Comment:
Shouldn't be `OfflineRetransmissionRepository` instead of `ZookeeperOfflineRetransmissionRepository`? (to be consistent with the fields above)